src/CloudNative.CloudEvents/CloudEvent.cs

```diff
@@ -131,6 +161,7 @@ public CloudEvent(CloudEventsSpecVersion specVersion, IEnumerable<CloudEventAttr
                     return;
                 }
                 // TODO: We could convert the attribute value here instead? Or is that a bit too much "magic"?
+                // TODO: Should we validate the value against the 
```

Comment:
You trailed off there 😄

Reply:
Whoops, yes. I think that was because I stopped writing the comment and implemented validation instead :) Deleted in a new commit.